templates/org/settings/secrets.tmpl

```diff
@@ -0,0 +1,96 @@
+{{template "base/head" .}}
+<div class="page-content organization settings webhooks">
+	{{template "org/header" .}}
+	<div class="ui container">
+		<div class="ui grid">
+			{{template "org/settings/navbar" .}}
+			<div class="ui twelve wide column content">
+				{{template "base/alert" .}}
+				<h4 class="ui top attached header">
+					{{.locale.Tr "repo.settings.secrets"}}
+					<div class="ui right">
+						<div class="ui primary tiny show-panel button" data-panel="#add-secret-panel">{{.locale.Tr "repo.settings.add_secret"}}</div>
+					</div>
+				</h4>
+				<div class="ui attached segment">
+					<div class="{{if not .HasError}}hide{{end}} mb-4" id="add-secret-panel">
+						<form class="ui form" action="{{.Link}}?act=secret" method="post">
+							{{.CsrfTokenHtml}}
+							<div class="field">
+								{{.locale.Tr "repo.settings.secret_desc"}}
+							</div>
+							<div class="field {{if .Err_Title}}error{{end}}">
+								<label for="title">{{.locale.Tr "repo.settings.secret_key"}}</label>
+								<input id="key-title" name="title" value="{{.title}}" autofocus required>
+							</div>
+							<div class="field {{if .Err_Content}}error{{end}}">
+								<label for="content">{{.locale.Tr "repo.settings.secret_content"}}</label>
+								<textarea id="key-content" name="content" placeholder="{{.locale.Tr "repo.settings.secret_value_content_placeholder"}}" required>{{.content}}</textarea>
+							</div>
+							<div class="field">
+								<div class="ui checkbox {{if .Err_PullRequestRead}}error{{end}}">
+									<input id="pull_request_read" name="pull_request_read" class="hidden" type="checkbox" value="1">
+									<label for="is_writable">
+										{{.locale.Tr "repo.settings.pull_request_read"}}
+									</label>
```

Comment:
Something doesn't add up with the `for`-parts of the labels here: Since when do they accept `name`? Don't they only accept `id`?